api/v1alpha4/condition_consts.go

```diff
@@ -201,3 +201,14 @@ const (
 	// from making any further remediations.
 	TooManyUnhealthyReason = "TooManyUnhealthy"
 )
+
+// Conditions and condition Reasons for  MachineDeployments
+
+const (
+	// MachineDeploymentAvailableCondition means the MachineDeployment is available, that is, at least the minimum available
+	// machines required (i.e. Spec.Replicas-MaxUnavailable when MachineDeploymentStrategyType = RollingUpdate) are up and running for at least minReadySeconds.
+	MachineDeploymentAvailableCondition ConditionType = "Available"
+
+	// WaitingForAvailableMachinesReason reflects the fact that the required minimum number of machines for a machinedeployment are not available.
```

Comment:
nit: Most other reason comments also add the **Severity** of the condition associated with the reason. Seems like a good idea to follow that practice.

Reply:
Done.